### PR TITLE
Adding javadoc for some classes

### LINKED ISSUE #233: Missing class JavaDocs
Module guidelines require us to have these, but for some reason the Checkstyle does not check for this.

I think for completeness we should add the doc for the class we wrote or maybe 1 person can add all in 1 PR idk.

# Missing class javadocs
 - ImportCommand.java
 - TagCommand.java
 - SummaryDisplay.java
 - UntagCommand.java
 - UndoCommand.java

### PR DESCRIPTION
**Pull request by aidoxe-123**
close #233

### CODE REVIEW

File: src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java [hunk 1]
```diff
@@ -11,14 +11,17 @@
 import jimmy.mcgymmy.model.ReadOnlyMcGymmy;
 import jimmy.mcgymmy.storage.JsonMcGymmyStorage;
 
+/**
+ * Changes the data in mcgymmy to the data in the specified file.
+ * File must be available and in correct format
```
Comment (Jh123x) [inline src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java]: Would be better if it is File must exist.

File: src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java [hunk 2]
```diff
@@ -11,14 +11,17 @@
...
+ */
 public class ImportCommand extends Command {
 
     public static final String COMMAND_WORD = "import";
     public static final String SHORT_DESCRIPTION = "Import a McGymmy Save file";
 
     public static final String MESSAGE_IMPORT_FOOD_SUCCESS = "Imported %s";
     public static final String MESSAGE_IMPORT_FOOD_FAILURE = "File is invalid";
-    public static final String FILE_CONSTRAINTS = "Please select a valid .json file";
```
Comment (Jh123x) [inline src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java]: This is updated in #237 as FILE_CONSTRAINTS is more specific, it replaced the MESSAGE_IMPORT_FOOD_FAILURE instead. But that is for another pr x.x

File: src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 import jimmy.mcgymmy.model.Model;
 
+/**
+ * Undo the previous action cause by other commands (except help).
```
Comment (Jh123x) [inline src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java]: Maybe it is better to be Undo the previous action that caused changes to ModelManager?

**Review by Jh123x — CHANGES_REQUESTED** [review]
Mostly LGTM.

File: src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java (hunk 2 above)
Comment (aidoxe-123) [reply src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java]: Ok I put it back

File: src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java (hunk 3 above)
Comment (aidoxe-123) [reply src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java]: Ok I've changed it already

File: src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java (hunk 1 above)
Comment (aidoxe-123) [reply src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java]: Fixed

**Comment by aidoxe-123** [comment]
Close since overlaps with #235

**Pull request closed without merging**